python_scripts/cross_validation_validation_curve.py

```diff
@@ -138,10 +130,28 @@
 # testing error is minimal, and this is what really matters. This is the
 # best compromise we could reach by just tuning this parameter.
 #
+# Be aware that looking at the mean errors is quite limiting. We should also
+# look at the standard deviation to assess the dispersion of the score. We
+# can repeat the same plot as before but this time, we will had some
+# information to show the standard deviation of the errors as well.
+
+# %%
+plt.errorbar(max_depth, train_errors.mean(axis=1),
+             yerr=3 * train_errors.std(axis=1), label='Training error')
+plt.errorbar(max_depth, test_errors.mean(axis=1),
+             yerr=3 * test_errors.std(axis=1), label='Testing error')
```

Comment:
Maybe just use 1 std for the error bars. 3 is kind of arbitrary and does not reflect what is written in the paragraph above.

Reply:
The error bar will be too small then. Maybe updating the test would be enough?

Reply:
Why too small? If the standard deviations are small, they are small, no?